spec/array_column_spec.cr

```diff
@@ -0,0 +1,18 @@
+require "./spec_helper"
+
+private class BucketQuery < Bucket::BaseQuery
+end
+
+describe "Array Columns" do
+  it "fails when passing a single value to an array query" do
+    bucket = BucketBox.new.numbers([1, 2, 3]).create
```

Comment:
Could you create an issue to tackle this later? I think it’ll require reworking the criteria methods a bit and will probably be a pretty big refactor